environment.yml

```diff
@@ -11,4 +11,5 @@ dependencies:
   - pip:
     - pytest
     - pytest-cov
+    - pytest-pythonpath
```

Comment:
I thought about using it for modules in development which were not on the PYTHONPATH. But, since the whole folder is now importable, it can be removed. Done.